### DISCUSSION TITLE
how train model using word embedding as input

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1710] [General Q&A]
I have large data of wav and csv text single person. I want to train TTS model for other language(not English) which model should i use and I want to know how use word embedding as input. I hope you help me.

thanks in advance 🙏🙏

**Comment by erogol** [comment]
We don't support that yet. You need to code yourself.

**Reply by omkarade** [reply]
how give input to model

**Reply by omkarade** [reply]
I only have wav files and respective text csv file. I don't have phonemes of my language .
